src/identity/privateHashing.ts

```diff
@@ -130,17 +127,27 @@ function* getPhoneHashPrivate(e164Number: string, selfPhoneHash?: string) {
   Logger.debug(TAG, '@fetchPrivatePhoneHash', 'Blinding factor', blindingFactor)
   const blsBlindingClient = new ReactBlsBlindingClient(networkConfig.odisPubKey, blindingFactor)
   try {
-    return yield call(
-      OdisUtils.PhoneNumberIdentifier.getPhoneNumberIdentifier,
+    const identifierHashDetails: IdentifierHashDetails = yield call(
+      OdisUtils.Identifier.getObfuscatedIdentifier,
       e164Number,
+      IdentifierPrefix.PHONE_NUMBER,
       accountAddress,
       authSigner,
       serviceContext,
       blindingFactor,
-      selfPhoneHash,
       DeviceInfo.getVersion(),
-      blsBlindingClient
+      blsBlindingClient,
+      undefined,
+      undefined,
+      CombinerEndpointPNP.LEGACY_PNP_SIGN
     )
+    const phoneNumberHashDetails: PhoneNumberHashDetails = {
```

Comment:
making this `getObfuscatedIdentifier` backwards compatible with `getPhoneNumberIdentifier` with the least amount of changes...